src/ensembl/src/shared/components/accordion/components/AccordionItemButton.tsx

```diff
@@ -1,30 +1,43 @@
 import React from 'react';
+import noop from 'lodash/noop';
+import classNames from 'classnames';
+
 import { InjectedButtonAttributes } from '../helpers/AccordionStore';
 import { DivAttributes } from '../helpers/types';
-import classNames from 'classnames';
 import { Consumer as ItemConsumer, ItemContext } from './ItemContext';
 
 import defaultStyles from '../css/Accordion.scss';
 
 type Props = DivAttributes & {
   extendDefaultStyles: boolean;
   toggleExpanded(): void;
+  disabled?: boolean;
 };
 
 export const AccordionItemButton = (props: Props) => {
-  const { className, extendDefaultStyles, toggleExpanded, ...rest } = props;
+  const {
+    className,
+    extendDefaultStyles,
+    toggleExpanded,
+    disabled,
+    ...rest
+  } = props;
 
   let styles = className;
 
   if (extendDefaultStyles) {
-    styles = classNames(defaultStyles.accordionButtonDefault, className);
+    styles = classNames(
+      defaultStyles.accordionButtonDefault,
+      { [defaultStyles.accordionButtonDefaultDisabled]: props.disabled },
```

Comment:
Any reason why the name ends in DefaultDisabled, and not just Disabled?

Reply:
I was thinking that the disabled button has a default style which can be overridden. I have renamed it to be just Disabled as it sounds better.